requirements.txt

```diff
@@ -62,4 +62,5 @@ git+https://github.com/luisza/django-gentelella-widgets.git#egg=djgentelella
 django-picklefield==2.1.1
 Markdown==3.2.2
 coreapi==2.3.3
-future==0.18.2
\ No newline at end of file
+future==0.18.2
+pdfkit==0.6.1
```

Comment:
pdfkit==0.6.1 is not a requirement

Reply:
I forgot to remove the library and also I don't know if WeasyPrint can be removed?

Reply:
Yes, the idea of this PR is remove WeasyPrint

Reply:
Great.